bucket/zoom.json

```diff
@@ -0,0 +1,31 @@
+{
+    "version": "4.6.7",
+    "description": "Video and audio conferencing, chat, and webinars",
+    "homepage": "https://zoom.com.cn/",
+    "license": {
+        "identifier": "Shareware",
```

Comment:
Which part of the agreement indicates shareware?

Reply:
Sorry, I'm not familiar with these licenses, since it can be used free, I guess it's license is Shareware. If not right, maybe Proprietary.